softwarereview_author.Rmd

```diff
@@ -4,23 +4,35 @@
 This concise guide presents the software peer review process for you as a package author.
 ```
 
+## Prior to Submission
```

Comment:
Maybe submission planning should be an item even before this one?

Reply:
ah that's the actual content. Maybe the title just needs to contain "planning"

Reply:
Rephrased to "Planning a submission" - thanks